### PR TITLE
Discourage controller specs in README

### PR DESCRIPTION
**Pull request by rlue**
Per issue #1838.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,57 +160,12 @@ end
...
-      post1, post2 = Post.create!, Post.create!
-      get :index
-
-      expect(assigns(:posts)).to match_array([post1, post2])
-    end
-  end
-end
-```
-
-For more information, see [cucumber scenarios for controller
```
Comment (xaviershay) [inline README.md]: Should probably keep this link around

Comment (rlue) [inline README.md]: Oops, accidentally closed previous review.

Link restored!

**Comment by samphippen** [comment]
LGTM. @xaviershay your feedback has been addressed so I'm going to merge.

**Comment by xaviershay** [comment]
thanks, sorry I missed the update!

**Pull request merged**